tests/components/intellifire/test_climate.py

```diff
@@ -0,0 +1,38 @@
+"""Test climate."""
+
+from unittest.mock import patch
+
+from freezegun import freeze_time
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, snapshot_platform
+
+
+@freeze_time("2021-01-01T12:00:00Z")
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_all_sensor_entities(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    mock_config_entry_current: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+    mock_fp,
+) -> None:
+    """Test all entities."""
+    with (
+        patch("homeassistant.components.intellifire.PLATFORMS", [Platform.CLIMATE]),
+        patch(
+            "intellifire4py.unified_fireplace.UnifiedFireplace.build_fireplace_from_common",
+            return_value=mock_fp,
+        ),
```

Comment:
Please create a mock for this. Don't mock at library level but where we use the library